service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -62,6 +68,23 @@ public void sendNotificationToHmctsAdmin(SubmittedCaseEvent event) {
         sendNotification(HMCTS_COURT_SUBMISSION_TEMPLATE, email, parameters, reference);
     }
 
+    @EventListener
+    public void sendNotificationForC2Upload(final C2UploadNotifyEvent caseEvent) {
+        UserDetails userDetails = caseEvent.getUserDetails();
+        List<String> roles = userDetails.getRoles();
+
+        if (!roles.containsAll(UserRole.HMCTS_ADMIN.getRoles())) {
```

Comment:
I would probably just check contain HMCTS_ADMIN. No reason to check others.

Reply:
yes we could, but the roles where returned as List, why we moved it into a UserRole enum